src/components/tabBars.js

```diff
@@ -168,7 +169,8 @@ export function SimpleTabBar({
   children,
   ...props
 }) {
-  const tabIds = _.map(useUniqueId, _.range(0, tabs.length));
```

Comment:
This violated the "rules of hooks" and would cause an error if the number of tabs changed between renders.

https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks